### DISCUSSION TITLE
Problems using Sorted containers on Leetcode

### DISCUSSION THREAD

**Discussion by harshraj22** [harshraj22/problem_solving#8] [General]
Unlike `c++`, Python's `set` and `dict` are not sorted.
Leetcode allows to use the library `SortedContainers`. It contains implementations for set and dict that remain sorted even after insertion and removal. Below are a few problems to be practiced in order to get familiar with the library:

- [ ] [exam room](https://leetcode.com/problems/exam-room/)
- [ ] [best-sightseeing-pair](https://leetcode.com/problems/best-sightseeing-pair/)
- [ ] [Largest rectangle in histogram](https://leetcode.com/problems/largest-rectangle-in-histogram/):  [cpp sol](https://github.com/harshraj22/problem_solving/blob/75a588ee78bf57dae24e0ebde53c3c6836787eb6/solution/leetcode/84.cpp)
- [ ] [Maximum Frequency Stack](https://leetcode.com/problems/maximum-frequency-stack/)
- [ ] [My Calendar 1](https://leetcode.com/problems/my-calendar-i/)
- [ ] [furthest-building-you-can-reach](https://leetcode.com/problems/furthest-building-you-can-reach/)
- [ ] [jump-game-vi](https://leetcode.com/problems/jump-game-vi/)
- [ ] [maximum-subarray](https://leetcode.com/problems/maximum-subarray/)

**Comment by Ainz07** [comment]
Have you encountered any more similar questions?
